### PR TITLE
Modify container status service

### PR DESCRIPTION
**Pull request by GonzaNunezCano**
### Brief Description
Add a service to allow a user to modify the container status. 
### Related card or ticket
https://github.com/Pis-moove-it/reciclando-backend/issues/31

### CODE REVIEW

File: app/controllers/containers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class ContainersController < BaseController
```
Comment (fbalarini) [inline app/controllers/containers_controller.rb:2]: Add a `TODO` here to remind us to authenticate this services, makes sense?

File: app/controllers/containers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  def update
+    if container.update(container_params)
+      head :ok
```
Comment (fbalarini) [inline app/controllers/containers_controller.rb]: I think we should return the updated container, mobile team will need it

File: spec/controllers/containers_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      it 'does return success' do
+        update_container_call(container.id, updated_container.status)
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when updating non-existent containers' do
+      it 'does return not found' do
+        update_container_call(invalid_id, updated_container.status)
+        expect(json_response['error_code']).to eq 3
```
Comment (fbalarini) [inline spec/controllers/containers_controller_spec.rb:25]: Check http status to equal 401

File: spec/controllers/containers_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      end
+    end
+
+    context 'when updating containers with wrong values' do
+      it 'does return bad request, nil status value' do
+        update_container_call(container.id, nil)
+        expect(json_response['error_code']).to eq 1
```
Comment (fbalarini) [inline spec/controllers/containers_controller_spec.rb:33]: same here but with 400

File: spec/controllers/containers_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      end
+      it 'does return internal error, invalid status value' do
+        update_container_call(container.id, 'invalid_value')
+        expect(json_response['error_code']).to eq 4
```
Comment (fbalarini) [inline spec/controllers/containers_controller_spec.rb:38]: same here

File: app/models/container.rb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+class Container < ApplicationRecord
+  enum status: %i[Ok Damaged Removed]
+  validates :status, :active, presence: true
```
Comment (fbalarini) [inline app/models/container.rb]: what do you think about adding a default value to `active`? I think that if we add `true` as a default value, creating a container is easier and better to understand. Afterwards, we will need to allow an admin to change this value if necessary.(changing it from true to false)

File: spec/controllers/containers_controller_spec.rb (hunk 3 above)
Comment (GonzaNunezCano) [reply spec/controllers/containers_controller_spec.rb:25]: Did you mean 404?

**Comment by fbalarini** [comment]
This PR is no more necessary, every change should be in `Modify container status admin`

**Pull request closed without merging**